### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by agrampa**

### CODE REVIEW

File: lab-allie/test/test.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        .get('/blah')
+        .send({})
+        .end((err, res) => {
+          expect(err).to.be.null;
+          expect(res.status).to.equal(400);
+        });
+        done();
+      });
+      
+      it('should respond with a 200 on proper request', done => {
```
Comment (JudyVue) [inline lab-allie/test/test.js:159]: No points lost, but remember to semantically write your ``it blocks`` to match with your expected result.

File: lab-allie/server.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+const server = module.exports = http.createServer(function(req, res) { 
+  req.url = url.parse(req.url); 
+  req.url.query = querystring.parse(req.url.query);
+  
+  if(req.method === 'POST') {
+    if(req.url.pathname === '/cowsay') {
+      return bodyParser(req, function(err) {
+        if(err) console.error(err);
+        console.log(req.body.text);
```
Comment (JudyVue) [inline lab-allie/server.js]: **TODO FOR LOT POINTS**

Take out stray console.logs/comments.

File: lab-allie/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# COWSAY project
```
Comment (JudyVue) [inline lab-allie/readme.md:1]: Good README, but make sure to be more specific in future docs, i.e. tell a user how to install ``httpie`` on their computer and use the ``http`` commands in their terminal.

File: lab-allie/server.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+const server = module.exports = http.createServer(function(req, res) { 
+  req.url = url.parse(req.url); 
+  req.url.query = querystring.parse(req.url.query);
+  
+  if(req.method === 'POST') {
+    if(req.url.pathname === '/cowsay') {
+      return bodyParser(req, function(err) {
+        if(err) console.error(err);
+        let message = cowsay.say({text: req.body.text, f: 'goat'});
```
Comment (jurrish) [inline lab-allie/server.js:18]: woot. stretch :)

**Pull request merged**